syntax = "proto3";

package hello_world;

// Il servizio definisce i metodi che il client pu√≤ chiamare
service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

// Struttura del messaggio di richiesta
message HelloRequest {
  string name = 1;
}
// Struttura del messaggio di risposta
message HelloReply {
  string message = 1;
}
